test-apps/appui-test-app/connected/eslint.config.js

```diff
@@ -0,0 +1,44 @@
+const iTwinPlugin = require("@itwin/eslint-plugin");
+const prettierConfig = require("eslint-config-prettier");
+
+const customLanguageOptions = {
+  sourceType: "module",
+  parser: require("@typescript-eslint/parser"),
+  parserOptions: {
+    project: ["./tsconfig.backend.json", "./tsconfig.json"],
+    ecmaVersion: "latest",
+    ecmaFeatures: {
+      jsx: true,
+      modules: true,
+    },
+  },
+};
+
+module.exports = [
+  {
```

Comment:
add the entry for the uiConfig like in arun's comment

Reply:
uiConfig doesn't seem to be part of the older eslint config, so this addition would be entirely new correct? And I should add uiConfig to all configs inside `test-apps/appui-test-app`?

Reply:
Yes, to all of the test apps. For connected and standalone it would be the src/frontend folder. for appui-test-providers it looks like it would be the ui folder

Reply:
gotcha, all 3 packages should have uiConfig in their eslint config now